### ISSUE TITLE
Question about BLE data

### ISSUE THREAD

**Issue by rennsport** [calvinmclean/stagg-ekg-plus#6]
Hello! I had a question about the BLE data stream that comes from the kettle. Once I authenticate with `efdd0b3031323334353637383930313233349a6d`, I can successfully turn on/off the kettle and set the temperature, but I'm not quite sure how to read out the temperature from the stream of data. I've read the reddit post which is extremely helpful, but unfortunately I'm not seeing any payloads with the form `eedd0x`. Here is my payload log between two FFFFFFFF payloads with the kettle turned on:

```Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: FFFFFFFF
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: EFDD03
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: 6F016F01
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: EFDD02
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: 73017301
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: EFDD00
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: 000000
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: EFDD01
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: 010100
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: EFDD06
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: 000000
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: EFDD07
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: 000000
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: EFDD08
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: 010100
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: EFDD04
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: 00000000
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: EFDD05
Peripheral: FELLOW7132 didUpdateValueFor: 2A80 to: FFFFFFFF
```
Thanks in advanced!

**Comment by calvinmclean** [comment]
I haven’t worked in this in a long time so unfortunately it’s all pretty unfamiliar to me now. Maybe the post had a typo and it should be “EFDD0” which is present in your logs? Have you tried running this program with your kettle?

**Comment by rennsport** [comment]
Hi Calvin, sorry for the late follow-up. I figured out that the two temperatures are `6F016F01` and `73017301`. I have not tried running the program yet as I don't have a full-fledged Pi (or a similar board) to run it on. Instead I am attempting to write my own in MicroPython for one of the Pi PicoWs I have. I think I've figured out _most_ of it other than some strange edge cases which I attribute to MicroPython.
